.vscode/ltex.dictionary.en-US.txt

```diff
@@ -0,0 +1,60 @@
+aclass
```

Comment:
it his vscode config necessary?

Reply:
It is just a dictionary file for spelling checks that would just work for vscode users,
and would not have harmed anybody else.

Of course, I removed as per your comment: Correct spelling is not needed.